apps/web/lib/hooks/useSlots.ts

```diff
@@ -191,13 +198,20 @@ export const useSlots = (props: UseSlotsProps) => {
       eventLength,
       beforeBufferTime,
       afterBufferTime,
+      currentSeats: responseBody.currentSeats,
     };
     const filteredTimes = getFilteredTimes(filterTimeProps);
     // temporary
     const user = res.url.substring(res.url.lastIndexOf("/") + 1, res.url.indexOf("?"));
+    const { currentSeats } = responseBody;
     return filteredTimes.map((time) => ({
       time,
       users: [user],
+      // Conditionally add the booking info slots object if there is already a booking during that time
+      ...(currentSeats?.some((booking) => booking.startTime === time.toISOString()) && {
+        bookingInfo:
+          currentSeats[currentSeats.findIndex((booking) => booking.startTime === time.toISOString())],
+      }),
```

Comment:
When passing time slots to available times we also add the booking id and the count of attendees